### ISSUE TITLE
Application controls info on first visit.

### ISSUE THREAD

**Issue by MuhammadSaadQadeer** [pusher/react-slack-clone#58]
For a user who is viewing the application for the first time, an assistant component can be implemented to let user know about the controls of the application and its features.

**Comment by lukejacksonn** [comment]
I have seen this kind of thing done well on other apps (Slack themselves are pretty damn good at it themselves IIRC) but I have never attempted to code one up. I'd be up for hearing how you would go about implementing a feature like this!

**Comment by MuhammadSaadQadeer** [comment]
Yeah you are right, well I have pulled the code , I hope it wont take much time for me to understand the code and structure, once I am done that part , I would give this one an immediate try. And will share the mockups with you before heading in to the development.
